tests/integration/security/jwt_test.go

```diff
@@ -427,43 +395,23 @@ func TestIngressRequestAuthentication(t *testing.T) {
 							return to.Match(echo.InCluster(from.Config().Cluster))
 						}).
 						To(util.DestFilter(apps, ns.Name(), false)...).
-						Run(func(t framework.TestContext, from echo.Instance, to echo.Target) {
-							callCount := 1
-							if t.Clusters().IsMulticluster() {
-								// so we can validate all clusters are hit
-								callCount = util.CallsPerCluster * to.WorkloadsOrFail(t).Len()
-							}
-							for _, c := range cases {
-								t.NewSubTest(c.name).Run(func(t framework.TestContext) {
-									opts := echo.CallOptions{
-										To: to,
-										Port: echo.Port{
-											Name: "http",
-										},
-										Count: callCount,
-									}
-
-									// Apply any custom options for the test.
-									c.customizeCall(&opts)
-
-									from.CallOrFail(t, opts)
-								})
-							}
-						})
+						RunCases(cases)
 				}
 			}
 
-			t.NewSubTest("in-mesh-authn").Run(newTest("testdata/requestauthn/ingress.yaml.tmpl", []testCase{
+			t.NewSubTest("in-mesh-authn").Run(newTest("testdata/requestauthn/ingress.yaml.tmpl", []echotest.TestCase{
 				{
-					name: "in-mesh-with-expired-token",
-					customizeCall: func(opts *echo.CallOptions) {
+					Name: "in-mesh-with-expired-token",
+					CustomizeCall: func(_ framework.TestContext, opts *echo.CallOptions) {
```

Comment:
why does it take a TestContext? I didn't see any usages of this (but may have missed them)